doc/user_manual/existing_interfaces.tex

```diff
@@ -464,6 +464,45 @@ \subsubsection{Models}
 </Models>
 \end{lstlisting}
 
+An additional feature of the RELAP5 Code interface is the possibility to specify operation based on the value of user-inputted
+cards. For example, let's assume the values in cards 1180801:2 and 1180802:2 must come from a calculation based on sampled variables (e.g. 20100154:2 and 20100155:2), the user can specify the following XML node:
+
```

Comment:
why dont use the functions on the sampler directly?

Reply:
Because the function can not access to the values in the input files of the driven model.